packages/js/src/containers/AdvancedSettings.js

```diff
@@ -13,23 +13,26 @@ export default compose( [
 			getAdvanced,
 			getBreadcrumbsTitle,
 			getCanonical,
+			getTimestamp,
```

Comment:
I would also consider prefixing with `wordproof` here, since the component is not WordProof specific.